### PR TITLE
[Logs] Added a support for yaml files with .yml extension

### PR DESCRIPTION
**Pull request by ajacquemot**
### What does this PR do?

Added a support for YAML files with `.yml` extension.

### CODE REVIEW

File: pkg/logs/config/integration_config.go [hunk 1]
```diff
@@ -31,8 +31,13 @@ const (
 	MultiLine      = "multi_line"
 )
 
+// Valid integration config extensions
+const (
+	yamlExtention = ".yaml"
```
Comment (masci) [inline pkg/logs/config/integration_config.go]: typo: s/yamlExtention/yamlExtension

**Review by mnshdw — APPROVED** [review]
LGTM

**Comment by tmichelet** [comment]
@ajacquemot could we use the agent's logic for that instead?

**Comment by ajacquemot** [comment]
You're right, I just wanted to use our old solution since we are not plugged to `autodiscovery` yet.
I created a ticket for this purpose: https://trello.com/c/Ds2YCuCS/189-use-auto-config, feel free to better the description :)

**Comment by tmichelet** [comment]
ok thank you

**Pull request merged**